SecretManagement.PleasantPasswordServer.Extension/Private/Invoke-LoginToPleasant.ps1

```diff
@@ -62,10 +77,11 @@ function Invoke-LoginToPleasant
     }
     else
     {
-        # Generate JSON token
-        $Token = (ConvertFrom-Json $JSON.Content).access_token
+        # Generate and store JSON token
+        $script:loginToken = ConvertFrom-Json $JSON.Content
+        $script:loginToken | Add-Member -MemberType NoteProperty -Name "expires" -value (get-date).AddSeconds($script:LoginToken.expires_in)
```

Comment:
Please rewrite like this
```powershell
$script:loginToken = [PSCustomObject]@{
    access_token = (ConvertFrom-Json $JSON.Content).access_token
    expires      = (Get-Date).AddSeconds($script:LoginToken.expires_in)
}
```
It is the newer and more modern way

Reply:
I am still learning, thanks for your feedback. 

I have amended it to the below to prevent needing to call `ConvertFrom-Json` multiple times.

```
# Generate and store JSON token
        $token = ConvertFrom-Json $JSON.Content

        $script:loginToken = [PSCustomObject]@{
            access_token = $token.access_token
            expires      = (Get-Date).AddSeconds($token.expires_in)
        }
```

Let me know your thoughts.